source/_integrations/mawaqit.markdown

```diff
@@ -0,0 +1,69 @@
+---
+title: MAWAQIT Prayer Times
+description: Instructions on how to integrate the MAWAQIT Prayer Times integration within Home Assistant.
+ha_category:
+  - Sensor
+ha_iot_class: cloud_polling
+ha_release: 2024.10.2
+ha_config_flow: true
+ha_domain: mawaqit
+ha_codeowners:
+  - '@MAWAQIT'
+  - '@moha-tah'
+  - '@yeyvo'
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+The MAWAQIT (`mawaqit`) integration displays the various prayer times for Muslims as sensors.
```

Comment:
```suggestion
The  MAWAQIT Prayer Times integration displays the various prayer times for Muslims as sensors.
```
In the header, the integration is called "MAWAQIT Prayer Times", but in the manifest, it is MAWAQIT. What is the correct name of the integration?